index.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const server = express();
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development)
+
+server.use(express.json())
+
+server.get('/api/projects', async(req, res) => {
+    try{
+        const projects = await db('projects');
+        res.status(200).json(projects)
+    }catch (error){
+        res.status(500).json(error)
+    }
+})
+
+server.get('/api/projects/:id', async(req, res) => {
+    try{
+        const project = await db('projects')
+        .where({ id: req.params.id })
+        .first()
+        .then(project => {
+            if (project) {
+                db('action')
+                .where({ project_id: req.params.id})
+                .then(action => {
```

Comment:
I'd have used actions in plural, since you are getting an array back, other than that excellent job